src/index.js

```diff
@@ -13,5 +15,13 @@ const MaterialIcon = ({ icon, size, invert, inactive, color, style }) => {
         <i className={`material-icons ${sizeMapped} ${defaultColor} ${inactiveColor}`} style={styleOverride} >{icon}</i>
     )
 }
+
+MaterialIcon.propTypes = {
+  icon: PropTypes.string.isRequired,
+  size: PropTypes.any,
```

Comment:
Use `PropTypes.oneOfType([PropTypes.number, PropTypes.string])`

Reply:
Oh that sounds like a good idea, didn't know it worked that way. Will make the changes!
@rafaelklaessen thank you